src/main/webapp/resources/js/pages/launch/references/UploadReferenceFile.jsx

```diff
@@ -31,7 +31,7 @@ export function UploadReferenceFile({ form }) {
         form.setFieldsValue({ reference: info.file.response.files[0].id });
       } else if (status === "error") {
         notification.error({
-          message: i18n("UploadReferenceFile.error", info.file.name),
+          message: response.error
```

Comment:
Can you add a test that shows that when a file with bad characters is uploaded this message is displayed (and has the expected content).

Reply:
As discussed this test won't be added here